components/home/DashboardButton.tsx

```diff
@@ -14,16 +14,18 @@ interface DashboardButtonProps {
 export default function DashboardButton({
   iconName,
   iconLibrary,
-  route = '/Deposit',
+  route,
 }: DashboardButtonProps) {
   const colorScheme = useColorScheme();
   const onPress = () => {
```

Comment:
onPress function is now legacy. Can you remove  ? thank you :)

Reply:
@GILREICH1  Pushed requested changes, please review.